learning_paths/api/v1/views.py

```diff
@@ -393,3 +395,42 @@ def post(self, request, *args, **kwargs):
             },
             status=status.HTTP_201_CREATED,
         )
+
+
+class LearningPathCourseEnrollmentView(APIView):
+    """API View to enroll a user in a course that's part of a learning path."""
+
+    permission_classes = [IsAuthenticated, IsAdminOrSelf]
+
+    def _get_learning_path(self, learning_path_key_str: str) -> LearningPath:
```

Comment:
Isn't this the same as the `_get_learning_path` method defined in `LearningPathEnrollmentView`? Can't we reuse that?

Reply:
Ah okay I saw that you had a step in get method of `LearningPathEnrollmentView` to filter it based on enrollment and was wondering if we can't do that here too. Perhaps we can rename this method to something like `get_enrolled_learning_path` instead for clarity?